src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -381,6 +382,49 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should retire a concept', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: sampleRetiredConcept,
+      });
+    });
+    const store = mockStore({ payload: {} });
+    return store.dispatch(retireConcept(sampleConcept.url, { retired: true })).then((result) => {
+      expect(result.retired).toEqual(true);
+    });
+  });
+
+  it('should unretire a concept', () => {
```

Comment:
When should unretire a concept happen?

Reply:
This action is triggered when the `unretire` button is clicked

Reply:
I have updated the message accordingly.